processor/eval/ledger_for_evaluator.go

```diff
@@ -139,6 +139,16 @@ func (l LedgerForEvaluator) GetAppCreator(indices map[basics.AppIndex]struct{})
 	return res, nil
 }
 
+// LookupKv is part of go-algorand's indexerLedgerForEval interface.
+func (l LedgerForEvaluator) LookupKv(round basics.Round, key string) (*string, error) {
+	// a simple pass-thru to the go-algorand ledger
+	val, err := l.Ledger.LookupKv(round, key)
+
+	fmt.Printf("\nindexer::LedgerForEvaluator.LookupKv(round=%d, key=%s) --> (%v, %v)", round, key, val, err)
```

Comment:
TODO: don't merge with this print statement

Reply:
And, IMO, just `return l.Ledger.LookupKv(round, key)`

Reply:
resolving